node/src/BaseClient.ts

```diff
@@ -6314,12 +6340,17 @@ export class BaseClient {
      * ```
      */
     public async lcsIdx(
-        key1: string,
-        key2: string,
-        options?: { withMatchLen?: boolean; minMatchLen?: number },
+        key1: GlideString,
+        key2: GlideString,
+        options?: {
+            withMatchLen?: boolean;
+            minMatchLen?: number;
+        } & DecoderOption,
     ): Promise<Record<string, (number | [number, number])[][] | number>> {
+        // TODO: convert Record return type to Object array
```

Comment:
I just double checked - the function returns 2 fixed key names and numbers. You can hardcode string decoder there.